data/configd-include.json

```diff
@@ -27,6 +27,7 @@
 "nat.py",
 "nat64.py",
 "nat66.py",
+"nat_cgnat.py",
```

Comment:
This is obsolete now, the file was removed in 

* https://github.com/vyos/vyos-1x/pull/3727

and is now auto generated

Reply:
Fixed!